include/mbedtls/entropy.h

```diff
@@ -117,21 +118,21 @@ mbedtls_entropy_source_state;
  */
 typedef struct mbedtls_entropy_context
 {
-    int accumulator_started; /* 0 after init.
+    int MBEDTLS_PRIVATE(accumulator_started); /* 0 after init.
                               * 1 after the first update.
                               * -1 after free. */
 #if defined(MBEDTLS_ENTROPY_SHA512_ACCUMULATOR)
-    mbedtls_sha512_context  accumulator;
+    mbedtls_sha512_context  MBEDTLS_PRIVATE(accumulator);
 #else
     mbedtls_sha256_context  accumulator;
```

Comment:
Yes, it should be wrapped. The script is based on preprocessing the source with a specific configuration, so it would either find one or the other occurrence. That's one of the reasons I'd like it to be idempotent @mstarzyk-mobica : then we can run it in multiple configurations.